Makefile

```diff
@@ -136,6 +138,30 @@ install-kernel:
 	@bin/container system stop || true
 	@bin/container system start --enable-kernel-install $(SYSTEM_START_OPTS)
 
+.PHONY: coverage
+coverage: init-block
+	@echo Ensuring apiserver stopped before the CLI integration tests...
+	@bin/container system stop && sleep 3 && scripts/ensure-container-stopped.sh
+	@bin/container system start $(SYSTEM_START_OPTS) && \
+	echo "Starting unit tests" && \
+	{ \
+		exit_code=0; \
+		$(SWIFT) test --no-parallel --enable-code-coverage -c $(BUILD_CONFIGURATION) $(SWIFT_CONFIGURATION) || exit_code=1 ; \
+		echo Ensuring apiserver stopped after the CLI integration tests ; \
+		scripts/ensure-container-stopped.sh ; \
+		echo Generating code coverage report... ; \
+		xcrun llvm-profdata merge -sparse $(COV_DATA_DIR)/*.profraw -o $(COV_DATA_DIR)/default.profdata ; \
+		xcrun llvm-cov show --compilation-dir=`pwd` \
+			-instr-profile=$(COV_DATA_DIR)/default.profdata \
+			--ignore-filename-regex=".build/" \
+			--ignore-filename-regex=".pb.swift" \
+			--ignore-filename-regex=".proto" \
+			--ignore-filename-regex=".grpc.swift" \
+			$(BUILD_BIN_DIR)/ContainerPackageTests.xctest/Contents/MacOS/ContainerPackageTests > $(COV_REPORT_FILE) ; \
```

Comment:
Why are we referencing xctest here? This doesn't work for me locally

Reply:
Mostly b/c this was the path for build output after running tests (make test) on my system. Maybe we can chat through this one in realtime, or others can TAL on their platform for what's unique here?